components/StoryListItem/StoryListItem.tsx

```diff
@@ -49,14 +104,48 @@ const StoryListItem: React.FC<StoryListItemProps> = ({ story }) => {
             <ul className="mt-4 sm:mt-0 space-x-2">
               {story.topics.map((topic: Topic) => (
                 <li
-                  className="text-xs text-white bg-el-paso inline-block rounded-full py-1 px-2"
+                  className="text-xs text-white bg-el-paso inline-block rounded-full py-1 px-4 h-7 inline-flex items-center"
                   key={topic.id}
                 >
                   {topic.name}
                 </li>
               ))}
             </ul>
           )}
+
+          {Object.keys(reactionOptionData).length > 0 && (
```

Comment:
Similar to the comment above, it might be nice if the API would give us something that reflects more closely the UI that we're going to need to build.

Reply:
Another thought that I had was that if we ended up needing to cache the aggregate values that we could use a JSON column that reflects the UI.